### PR TITLE
CTOR-1178 : Add unsupported model for hardware-storage-qnap-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Add unsupported model for hardware-storage-qnap-snmp
And update the documentation

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 1]
```diff
@@ -94,18 +108,18 @@ Here is the list of services for this connector, detailing all metrics linked to
...
 <TabItem value="Hardware-Global" label="Hardware-Global">
 
-| Metric name                  			   | Unit   |
-|:-----------------------------------------|:-------|
-| hardware.temperature.celsius             | C      |
-| hardware.disk.temperature.celsius        | C      |
-| hardware.mdisk.temperature.celsius       | C      |
-| hardware.fan.speed.rpm                   | rpm    |
-| hardware.powersupply.fan.speed.rpm       | N/A    |
+| Metric name                        | Unit |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:111]:
```suggestion
| Name                       | Unit |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+
+Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Qnap** 
+depuis la page **Configuration > Gestionnaire de connecteurs de supervision** :
+* [Base Pack](./base-generic.md)
+
 ## Contenu du pack
 
+### Modèles d'équipement supportés
+
+* EKI utilisant le système d'opération QTS
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:18]: c'est quoi un EKI?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
+
+### Modèles d'équipement non supportés
+
+* **Uniquement pour le service volume** : LAGMT utilisant le système d'opération QuTS hero (L'information "volume" n'est pas présente dans le fichier MIB du QuTS hero)
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:22]:
```suggestion
* **Uniquement pour le service volume** : LAGMT utilisant le système d'opération QuTS hero (L'information "volume" n'est pas présente dans le fichier MIB du QuTS hero).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md (hunk 2 above)
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:18]:
```suggestion
* EKI utilisant le système d'opération QTS.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 4]
```diff
@@ -50,31 +64,31 @@ Le connecteur apporte les modèles de service suivants
...
 #### Découverte de services
 
-| Nom de la règle                   | Description                                               |
-|:----------------------------------|:----------------------------------------------------------|
-| HW-Storage-Qnap-SNMP-Disk-Name    | Discover the disk partitions and monitor space occupation |
-| HW-Storage-Qnap-SNMP-Disk-Name_1  | Discover the disk partitions and monitor space occupation |
-| HW-Storage-Qnap-SNMP-Traffic-Name | Discover the disk partitions and monitor space occupation |
+| Nom de la règle                   | Description                                                                |
+|:----------------------------------|:---------------------------------------------------------------------------|
+| HW-Storage-Qnap-SNMP-Disk-Name    | Découvre les partitions du disque et superviser l'occupation de son espace |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:77]:
```suggestion
| HW-Storage-Qnap-SNMP-Disk-Name    | Découvre les partitions du disque et supervise l'occupation de son espace |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 5]
```diff
@@ -50,31 +64,31 @@ Le connecteur apporte les modèles de service suivants
...
+| HW-Storage-Qnap-SNMP-Disk-Name_1  | Découvre les partitions du disque et superviser l'occupation de son espace |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:78]:
```suggestion
| HW-Storage-Qnap-SNMP-Disk-Name_1  | Découvre les partitions du disque et supervise l'occupation de son espace |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 6]
```diff
@@ -50,31 +64,31 @@ Le connecteur apporte les modèles de service suivants
...
+| HW-Storage-Qnap-SNMP-Traffic-Name | Découvre les interfaces réseau et supervise le statut et l'utilisation     |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:79]:
```suggestion
| HW-Storage-Qnap-SNMP-Traffic-Name | Découvre les interfaces réseau et en supervise le statut et l'utilisation     |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 7]
```diff
@@ -1,12 +1,26 @@
...
+
 ## Pack assets
 
+### Supported equipment models
+
+* EKI using the QTS operating system
+
+### Unsupported equipment models
+
+* **Only for volume service** : LAGMT using the QuTS hero operating system (The volume information isn't present in the QuTS hero MIB file)
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:22]:
```suggestion
* **Only for volume service** : LAGMT using the QuTS hero operating system (The volume information isn't present in the QuTS hero MIB file).
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 8]
```diff
@@ -1,12 +1,26 @@
...
+
+The following monitoring connectors will be installed when you install the **Qnap** connector through the
+**Configuration > Monitoring Connector Manager** menu:
+* [Base Pack](./base-generic.md)
+
 ## Pack assets
 
+### Supported equipment models
+
+* EKI using the QTS operating system
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:18]:
```suggestion
* EKI using the QTS operating system.
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 9]
```diff
@@ -32,10 +46,10 @@ The connector brings the following service templates (sorted by the host templat
 
 | Service Alias        | Service Template                                 | Service Description                                                                          | Discovery  |
 |:---------------------|:-------------------------------------------------|:---------------------------------------------------------------------------------------------|:----------:|
-| Disk-Global          | HW-Storage-Qnap-Disk-Global-SNMP-custom          | Check the rate of free space on disks. For each check the mount point of disks will appear  | X          |
+| Disk-Global          | HW-Storage-Qnap-Disk-Global-SNMP-custom          | Check the rate of free space on disks. For each checks the mount point of disks will appear  | X          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:49]:
```suggestion
| Disk-Global          | HW-Storage-Qnap-Disk-Global-SNMP-custom          | Check the rate of free space on disks. For each check the mount point of disks will appear  | X          |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 10]
```diff
@@ -32,10 +46,10 @@ The connector brings the following service templates (sorted by the host templat
...
 | Pools                | HW-Storage-Qnap-Pools-SNMP-custom                | Check pools                                                                                  |            |
-| Traffic-Generic-Id   | HW-Storage-Qnap-Traffic-Generic-Id-SNMP-custom   | Check traffic of a network interface                                                        |            |
-| Traffic-Generic-Name | HW-Storage-Qnap-Traffic-Generic-Name-SNMP-custom | Check traffic of a network interface                                                        |            |
+| Traffic-Generic-Id   | HW-Storage-Qnap-Traffic-Generic-Id-SNMP-custom   | Check traffic of an network interface                                                        |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:51]:
```suggestion
| Traffic-Generic-Id   | HW-Storage-Qnap-Traffic-Generic-Id-SNMP-custom   | Check traffic of a network interface                                                        |            |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 11]
```diff
@@ -32,10 +46,10 @@ The connector brings the following service templates (sorted by the host templat
...
+| Traffic-Generic-Name | HW-Storage-Qnap-Traffic-Generic-Name-SNMP-custom | Check traffic of an network interface                                                        |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:52]:
```suggestion
| Traffic-Generic-Name | HW-Storage-Qnap-Traffic-Generic-Name-SNMP-custom | Check traffic of a network interface                                                        |            |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 12]
```diff
@@ -49,31 +63,31 @@ The connector brings the following service templates (sorted by the host templat
...
 
-| Rule name                         | Description                                               |
-|:----------------------------------|:----------------------------------------------------------|
-| HW-Storage-Qnap-SNMP-Disk-Name    | Discover the disk partitions and monitor space occupation |
-| HW-Storage-Qnap-SNMP-Disk-Name_1  | Discover the disk partitions and monitor space occupation |
-| HW-Storage-Qnap-SNMP-Traffic-Name | Discover the disk partitions and monitor space occupation |
+| Rule name                         | Description                                                   |
+|:----------------------------------|:--------------------------------------------------------------|
+| HW-Storage-Qnap-SNMP-Disk-Name    | Discover the disk partitions and monitor space occupation     |
+| HW-Storage-Qnap-SNMP-Disk-Name_1  | Discover the disk partitions and monitor space occupation     |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:77]: what's the difference between name and name_1?

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 13]
```diff
@@ -269,113 +283,113 @@ yum install centreon-plugin-Hardware-Storage-Qnap-Snmp
...
-| WARNING      | Warning threshold average CPU utilization                                                          | 80                |             |
-| CRITICAL     | Critical threshold average CPU utilization                                                         | 90                |             |
+| Macro        | Description                                                                                                                            | Default value     | Mandatory   |
+|:-------------|:---------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| WARNING      | Warning threshold average CPU utilization                                                                                              | 80                |             |
+| CRITICAL     | Critical  threshold average CPU utilization                                                                                            | 90                |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:289]:
```suggestion
| CRITICAL     | Critical threshold for average CPU utilization                                                                                            | 90                |             |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 14]
```diff
@@ -269,113 +283,113 @@ yum install centreon-plugin-Hardware-Storage-Qnap-Snmp
...
+| Macro        | Description                                                                                                                                                                                     | Default value     | Mandatory   |
+|:-------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTER       | Set the storage (number expected) example: 1, 2,... (empty means 'check all storage')                                                                                                           | .*                |             |
+| TRANSFORMDST | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run'  will replace all occurrences of 'dev' with 'run' |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:298]:
```suggestion
| TRANSFORMDST | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run' will replace all occurrences of 'dev' with 'run' |                   |             |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 15]
```diff
@@ -269,113 +283,113 @@ yum install centreon-plugin-Hardware-Storage-Qnap-Snmp
...
+| TRANSFORMSRC | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run'  will replace all occurrences of 'dev' with 'run' |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:299]:
```suggestion
| TRANSFORMSRC | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run' will replace all occurrences of 'dev' with 'run' |                   |             |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 16]
```diff
@@ -269,113 +283,113 @@ yum install centreon-plugin-Hardware-Storage-Qnap-Snmp
...
-| WARNINGOUT   | Thresholds                                                                                                                                            | 80                |             |
-| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                    |                   |             |
+| Macro        | Description                                                                                                                                            | Default value     | Mandatory   |
+|:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| INTERFACEID  | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored.  To filter on interface names, see --name |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:346]:
```suggestion
| INTERFACEID  | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored. To filter on interface names, see --name |                   |             |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 17]
```diff
@@ -269,113 +283,113 @@ yum install centreon-plugin-Hardware-Storage-Qnap-Snmp
...
-| WARNINGOUT    | Thresholds                                                                                                                                            | 80                |             |
-| EXTRAOPTIONS  | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                    |                   |             |
+| Macro         | Description                                                                                                                                            | Default value     | Mandatory   |
+|:--------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| INTERFACENAME | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored.  To filter on interface names, see --name |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:358]:
```suggestion
| INTERFACENAME | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored. To filter on interface names, see --name |                   |             |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 18]
```diff
@@ -269,113 +283,113 @@ yum install centreon-plugin-Hardware-Storage-Qnap-Snmp
...
-| WARNINGIN      | Thresholds                                                                                                                                                                                                          | 80                |             |
-| CRITICALOUT    | Thresholds                                                                                                                                                                                                          | 90                |             |
-| WARNINGOUT     | Thresholds                                                                                                                                                                                                          | 80                |             |
+| FILTER         | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored.  To filter on interface names, see --name                                                              | .*                |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:370]:
```suggestion
| FILTER         | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored. To filter on interface names, see --name                                                              | .*                |             |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md [hunk 19]
```diff
@@ -50,31 +64,31 @@ Le connecteur apporte les modèles de service suivants
...
-| Nom de la règle                   | Description                                               |
-|:----------------------------------|:----------------------------------------------------------|
-| HW-Storage-Qnap-SNMP-Disk-Name    | Discover the disk partitions and monitor space occupation |
-| HW-Storage-Qnap-SNMP-Disk-Name_1  | Discover the disk partitions and monitor space occupation |
-| HW-Storage-Qnap-SNMP-Traffic-Name | Discover the disk partitions and monitor space occupation |
+| Nom de la règle                   | Description                                                                |
+|:----------------------------------|:---------------------------------------------------------------------------|
+| HW-Storage-Qnap-SNMP-Disk-Name    | Découvre les partitions du disque et supervise l'occupation de son espace |
+| HW-Storage-Qnap-SNMP-Disk-Name_1  | Découvre les partitions du disque et supervise l'occupation de son espace |
+| HW-Storage-Qnap-SNMP-Traffic-Name | Découvre les interfaces réseau et en supervise le statut et l'utilisation     |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:79]:
```suggestion
| HW-Storage-Qnap-SNMP-Disk-Name    | Découvre les partitions du disque et supervise l'occupation de son espace |
| HW-Storage-Qnap-SNMP-Traffic-Name | Découvre les interfaces réseau et en supervise le statut et l'utilisation     |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md (hunk 12 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:77]:
```suggestion
| HW-Storage-Qnap-SNMP-Disk-Name    | Discover the disk partitions and monitor space occupation     |
```

Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/hardware-storage-qnap-snmp.md:77]:
```suggestion
| HW-Storage-Qnap-SNMP-Disk-Name    | Discover the disk partitions and monitor space occupation     |
```

**Pull request merged**